test/com/yetu/play/authenticator/routes/SecuredActionRouteSpec.scala

```diff
@@ -1,30 +1,41 @@
 package com.yetu.play.authenticator.routes
 
-import com.yetu.youtube.utils.BaseSpec
+import com.yetu.play.authenticator.utils.{BaseSpec, FakeGlobal}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 
 class SecuredActionRouteSpec extends BaseSpec {
 
-  val helloUrl = "/helloTestRoute"
-
-  s"non-authenticated GET request on $helloUrl" must {
+  s"non-authenticated GET request on ${FakeGlobal.routeHelloTest}" must {
     "return a 303 response" in {
-
-      val Some(response) = route(FakeRequest(GET, helloUrl))
+      val Some(response) = route(FakeRequest(GET, FakeGlobal.routeHelloTest))
       status(response) mustEqual (SEE_OTHER)
     }
-
   }
 
-  s"authenticated GET request on $helloUrl" must {
+  s"authenticated GET request on ${FakeGlobal.routeHelloTest}" must {
     "return a valid 200 response" in {
-
-      val response = getRequestAuthenticated(helloUrl)
+      val response = getRequestAuthenticated(FakeGlobal.routeHelloTest)
       status(response) mustEqual (OK)
       contentAsString(response) must include("hello")
     }
+  }
+
+  s"non-authenticated POST request on ${FakeGlobal.routeApiLogout}" must {
+    "return a valid 204 response when access token is removed" in {
+      val response = postRequestAuthenticated(s"${FakeGlobal.routeApiLogout}?access_token=${FakeGlobal.oauth2Info.accessToken}")
+      status(response) mustEqual (NO_CONTENT)
+    }
 
+    "return a valid 404 response when access token does not exist" in {
+      val response = postRequestAuthenticated(s"${FakeGlobal.routeApiLogout}?access_token=${FakeGlobal.oauth2Info.accessToken}")
+      status(response) mustEqual (NOT_FOUND)
```

Comment:
tests should be independent ideally. So instead of saving the identity and an access token inside the fake global object, do it for the respective tests. Otherwise you rely on the order of execution of tests, which is not guaranteed if they run in parallel.